src/Microsoft.Azure.SignalR.Management/Configuration/ServiceManagerOptions.cs

```diff
@@ -31,15 +31,10 @@ public class ServiceManagerOptions
         /// </summary>
         public IWebProxy Proxy { get; set; }
 
-        /// <summary>
-        /// Gets or sets a service endpoint of Azure SignalR Service.
-        /// </summary>
-        internal ServiceEndpoint ServiceEndpoint { get; set; }
-
         /// <summary>
         /// Sets multiple service endpoints of Azure SignalR Service.
         /// </summary>
-        internal ServiceEndpoint[] ServiceEndpoints { get; set; } //not ready for public use
+        public ServiceEndpoint[] Endpoints { get; internal set; } //todo not ready for public use
```

Comment:
`internal` first otherwise people can get it.

use EII to do so https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/interfaces/how-to-explicitly-implement-interface-members